mods/persistence/modules/fabrication/fabricator_protolathe.dm

```diff
@@ -0,0 +1,14 @@
+/obj/machinery/fabricator/protolathe
+	base_storage_capacity = list(
```

Comment:
This change should no longer be necessary (fabricators build their material capacity from their design list)

Reply:
unecessary file deleted